microsoft-edge/devtools-guide-chromium/javascript/reference.md

```diff
@@ -366,6 +366,41 @@ If you find yourself running the same debug code in the **Console** tool over an
 See [Run snippets of JavaScript on any webpage](./snippets.md).
 
 
+<!-- ====================================================================== -->
+## Use `# sourceURL` to name evaluated and inline code in the Sources tool
+
+The `# sourceURL` pragma, with syntax `//# sourceURL=my-assigned-file-name.js` or `/*# sourceURL=my-assigned-file-name.css*/`, is a special comment that makes it easier to work with evaluated and inline code in the **Sources** tool.
+
+When a block of code is run by the browser, DevTools can only give that block of code a name if it was loaded from an network resource. When running the following kinds of code, the **Sources** tool doesn't have a file name to display these blocks of code in the **Navigator** pane:
+
+* JavaScript code that's run by using the `eval()` function.
+* JavaScript code that's in a `<script>` tag.
+* CSS code that's in a `<style>` tag.
+
+Use the `# sourceURL` pragma to give a virtual name to the above kinds of code. For example:
```

Comment:
```suggestion
Use the `# sourceURL` pragma to give a virtual file name to the above kinds of code, to display that file name in the Sources tool. For example:
```
* reiterate purpose of defining virtual filename.  Actually, the section never really explicitly directly says that the _reason_ or effect of giving a (virtual) file name to block of code is, so that when you look at Sources > Nav pane > Files tab (or wherever in UI) -- specify exactly where in UI these virtual file names will appear.
* not just giving a "name" to the code, but specifically a virtual _file_ name.